API/GetWordLinkRecording.php

```diff
@@ -0,0 +1,51 @@
+<?php
+require_once('utils/Model.php');
+require_once('utils/Respond.php');
+
+use storyproducer\Respond;
+
+session_start();
+
+
+if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+    if (!(array_key_exists('PhoneId', $_POST) &&
+        array_key_exists('term', $_POST))) {
+        RespondWithError(400, 'This endpoint requires PhoneId and term');
+    }
+
+    $androidId = htmlspecialchars($_POST['PhoneId']);
+    $term = htmlspecialchars($_POST['term']);
+
+    $model = new Model();
+
+    # check if project exists for the given androidId
+    $projectId = $model->GetProjectId($androidId);
+    if (!$projectId) {
+        RespondWithError(404, "The requested term has not been registered.");
```

Comment:
Sorry. I gave you the wrong message. Can you change "requested term" to "requested PhoneId"? Feel free to push back when I make mistakes, too. :)